app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,90 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+:root{
+  --color-grey1: #777
```

```suggestion
  $color-grey1: #777
```
cho đoạn đặt biến color này sang file color.scss rồi import vào file này